src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -264,5 +271,26 @@ private Interop.procfs.ParsedStat GetStat()
             }
             return stat;
         }
+
+        /// <summary>Reads memory.limit_in_bytes cgroup data, if available.</summary>
+        /// <param name="limit">The read limit if available; otherwise, 0.</param>
+        /// <returns>true if the value could be read; otherwise, false.</returns>
+        private bool TryReadCGroupsMemoryLimit(out ulong limit)
+        {
+            const string CGroupMemoryLimitInBytesPath = "/sys/fs/cgroup/memory/memory.limit_in_bytes";
```

Comment:
It is more involved than using a fixed path. It can be mounted at different locations. Please take a look here on how it is done in FindCgroupPath in coreclr PAL:
https://github.com/dotnet/coreclr/blob/master/src/pal/src/misc/cgroup.cpp#L132-L310
I would also recommend to compute this path just once and cache it for perf reasons.

Reply:
Thanks.  Fixed.